src/SDKs/DataFactory/DataFactory.Tests/ScenarioTests/DataFactoryScenarioTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+using DataFactory.Tests.Utils;
+using Microsoft.Azure.Management.DataFactory;
+using Microsoft.Azure.Management.DataFactory.Models;
+using Microsoft.Rest.Azure;
+using System;
+using System.Linq;
+using Xunit;
+
+namespace DataFactory.Tests.ScenarioTests
+{
+    public class DataFactoryScenarioTests : ScenarioTestBase<DataFactoryScenarioTests>
+    {
+        [Fact]
+        [Trait(TraitName.TestType, TestType.Scenario)]
+        public void DataFactoryCrud()
+        {
+            var expectedFactory = new Factory(location: FactoryLocation);
+
+            Action<DataFactoryManagementClient> action = (client) =>
+            {
+                // create
```

Comment:
>// create [](start = 16, length = 9)

Minor comment - not a fan of comments in the code, it is self explanatory

Reply:
removed; these were originally comments for myself as I was creating the test